### PR TITLE
Extend and improve acceptance tests

### PR DESCRIPTION
**Pull request by relu**
Related to #7

### CODE REVIEW

**Review by phillebaba — CHANGES_REQUESTED** [review]
Thanks for the awesome work! I just have one small comment before we can merge :rose:

File: pkg/provider/data_sync_test.go [hunk 1]
```diff
@@ -1,29 +1,94 @@
...
+	resourceName := "data.flux_sync.main"
 	resource.ParallelTest(t, resource.TestCase{
-		Providers: testAccProviders,
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccSyncBasicDataSource,
+				// Without required target_path set
+				Config:      `data "flux_sync" "main" {  url = "ssh://git@example.com" }`,
```
Comment (phillebaba) [inline pkg/provider/data_sync_test.go:36]: I think we should break these one line strings into a separate constants if they contain variables. It makes them easier to edit later on.

**Review by phillebaba — APPROVED** [review]
LGTM

**Review by phillebaba — CHANGES_REQUESTED** [review]
Sorry, it seems like when rebasing something happened to the acceptance tests. Could you fix them?

My guess is that it has to do with the fields that were removed in the docs PR.

**Comment by relu** [comment]
> Sorry, it seems like when rebasing something happened to the acceptance tests. Could you fix them?
> 
> My guess is that it has to do with the fields that were removed in the docs PR.

Oops, my bad. Will push a fix.

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @relu 🍰

**Review by phillebaba — APPROVED** [review]

**Comment by phillebaba** [comment]
Great work @relu

**Pull request merged**